datasets/tmu_gfm_dataset/tmu_gfm_dataset.py

```diff
@@ -0,0 +1,115 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TMU-GFM-Dataset."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{yoshimura-etal-2020-reference,
+    title = "{SOME}: Reference-less Sub-Metrics Optimized for Manual Evaluations of Grammatical Error Correction",
+    author = "Yoshimura, Ryoma  and
+      Kaneko, Masahiro  and
+      Kajiwara, Tomoyuki  and
+      Komachi, Mamoru",
+    booktitle = "Proceedings of the 28th International Conference on Computational Linguistics",
+    month = dec,
+    year = "2020",
+    address = "Barcelona, Spain (Online)",
+    publisher = "International Committee on Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.coling-main.573",
+    pages = "6516--6522",
+    abstract = "We propose a reference-less metric trained on manual evaluations of system outputs for grammatical error correction (GEC). Previous studies have shown that reference-less metrics are promising; however, existing metrics are not optimized for manual evaluations of the system outputs because no dataset of the system output exists with manual evaluation. This study manually evaluates outputs of GEC systems to optimize the metrics. Experimental results show that the proposed metric improves correlation with the manual evaluation in both system- and sentence-level meta-evaluation. Our dataset and metric will be made publicly available.",
+}
+"""
+
+_DESCRIPTION = """\
+A dataset for GEC metrics with manual evaluations of grammaticality, fluency, and meaning preservation for system outputs. \
+More detail about the creation of the dataset can be found in Yoshimura et al. (2020).
+"""
+
+_HOMEPAGE = "https://github.com/tmu-nlp/TMU-GFM-Dataset"
+
+_LICENSE = ""
+
+_URLs = {
+    "default": "https://raw.githubusercontent.com/tmu-nlp/TMU-GFM-Dataset/main/tmu-gfm-dataset.csv",
+}
+
+
+class TMUGFMDataset(datasets.GeneratorBasedBuilder):
+    """TMU-GFM-Dataset."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "source": datasets.Value("string"),
+                "output": datasets.Value("string"),
+                "grammer": datasets.Value("string"),
+                "fluency": datasets.Value("string"),
+                "meaning": datasets.Value("string"),
```

Comment:
Could these ones be `datasets.Sequence(datasets.Value("int32"))` ?

After changing feature types don't forget to regenerate the dataset_infos.json file

Reply:
Sure! I'll change them to `datasets.Sequence(datasets.Value("int32"))`, and regenerate dataset_infos.json file.

Reply:
Fixed in e8c32c8.